contributors/design-proposals/local-storage-overview.md

```diff
@@ -348,24 +360,28 @@ All other partitions are exposed as persistent volumes. The PV interface allows
 
 # Open Questions & Discussion points 
 * Single vs split “limit” for storage across writable layer and logs
+    * Split allows for enforcement of hard quotas
+    * Single is a simpler UI
 * Local Persistent Volume bindings happening in the scheduler vs in PV controller
     * Should the PV controller fold into the scheduler
-* Supporting dedicated partitions for logs and volumes in Kubelet in addition to runtime overlay filesystem
-    * This complicates kubelet.Not sure what value it adds to end users.
-* Providing IO isolation for ephemeral storage
-    * IO isolation is difficult. Use local PVs for performance
-* Support for encrypted partitions
-* Do applications need access to performant local storage for ephemeral use cases? Ex. A pod requesting local SSDs for use as ephemeral scratch space.
-    * Typically referred to as “inline PVs” in kube land
-* Should block level storage devices be auto formatted to be used as file level storage?
-    * Flexibility vs complexity
-    * Do we really need this? 
+* Should block level storage devices be auto formatted to be used as file level storage instead of having the filesystems precreated by the admin?
+    * It would match behavior with GCE PD and EBS where the volume plugin will create the filesystem first.
+    * It can allow for more comprehensive (but slower) volume cleanup options.  The filesystem can be destroyed and then the partition can be zeroed.
+    * It limits the filesystem choices to those that k8 supports.
```

Comment:
There is no list of filesystem types in Kubernetes. It can support any fs, as long as mkfs.<fstype> is available on the host.